spec/views/shared/dialogs/_dialog_field_drop_down_list.html.haml_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe "shared/dialogs/dialog_field_drop_down_list.html.haml" do
```

Comment:
We don't need this spec anymore, since all of the logic is being handled by the helper method now, so I would say we can safely remove this file 😄 .